Makefile

```
@@ -1,7 +1,7 @@
 # Tunacode Development Makefile
 # Provides convenient shortcuts for common development tasks
 
-.PHONY: help dev-setup install run test lint clean
+.PHONY: help dev-setup install run test test-tmux lint clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)